### DISCUSSION TITLE
Exponential Radon Transform

### DISCUSSION THREAD

**Discussion by roflmaostc** [CERN/TIGRE#454] [General]
Hi!

I was wondering if you  ever considered implementing the [exponential radon transform](https://epubs.siam.org/doi/pdf/10.1137/0139029)?

The code base should be probably fine enough here, we would need to introduce something like a weight function which depends on the absolute coordinates, I guess.

Best,

Felix

**Comment by AnderBiguri** [comment]
Hi Felix, 
I never considered implementing it because TIGRE started as a CT toolbox (still _mostly_ is) so its been mostly application-driven, rather than maths-driven. I also am not really aware of the exponential radon transform. 
However, I have nothing against it being implemented in TIGRE. But I don't have time myself to put into it, so I would be happy to give pointers and suggestions if its something you want and are willing to implement.

**Comment by stephoog** [comment]
Hey! You can do the attenuated radon transform with [LEAP](https://github.com/llnl/leap). Try it!

**Reply by roflmaostc** [reply]
You work on TVAM with it :)?

I created [RadonKA.jl](https://github.com/roflmaostc/RadonKA.jl) in the meantime :laughing:
